frontend/src/features/workspace/components/CreateFormModal/CreateFormModalContent/CreateFormDetailsScreen.tsx

```diff
@@ -83,12 +86,25 @@ export const CreateFormDetailsScreen = (): JSX.Element => {
                 name="responseMode"
                 control={control}
                 render={({ field }) => (
-                  <FormResponseOptions {...field} showMrf={showMrf} />
+                  <WorkspaceRowsProvider>
+                    <FormResponseOptions
+                      {...field}
+                      showMrf={showMrf}
+                      hasSingpass={hasSingpass}
+                    />
+                  </WorkspaceRowsProvider>
                 )}
                 rules={{ required: 'Please select a form response mode' }}
               />
             </Skeleton>
             <FormErrorMessage>{errors.responseMode?.message}</FormErrorMessage>
+            {!hasSingpass || (
```

Comment:
supernit: `hasSingpass && (` will do the same thing!

Reply:
ty for the tip!